raphtory/src/python/graph/algorithm_result.rs

```diff
@@ -13,6 +14,19 @@ macro_rules! py_algorithm_result {
             >,
         );
 
+        impl Repr
+            for $crate::algorithms::algorithm_result::AlgorithmResult<
+                $rustKey,
+                $rustValue,
+                $rustSortValue,
+            >
+        {
+            fn repr(&self) -> String {
+                let algo_name = &self.algo_repr.algo_name;
+                format!("Algorithm Result: ({:?})", algo_name)
```

Comment:
I believe you need to use the other variables you added (i.e. num vertices and type)